packages/astro/src/vite-plugin-astro-server/plugin.ts

```diff
@@ -52,10 +54,10 @@ export default function createVitePluginAstroServer({
 			}
 
 			const loader = createViteLoader(viteServer);
-			const createExports = await loader.import('astro:app');
+			const createApp = await viteServer.environments.ssr.runner.import(ASTRO_DEV_APP_ID);
```

Comment:
Can we do this for clarity? I was confused for a sec because `createApp` has another meaning somewhere else
```suggestion
			const { default: createAstroServerApp } = await viteServer.environments.ssr.runner.import(ASTRO_DEV_APP_ID);
```